iteration/iteration_test.go

```diff
@@ -458,84 +450,51 @@ func (s *TestIterationRepository) TestExistsIteration() {
 
 func (s *TestIterationRepository) TestIsActive() {
 	t := s.T()
+	newSpace := space.Space{
+		Name:    testsupport.CreateRandomValidTestName("Space Exists"),
+		OwnerId: s.testIdentity.ID,
+	}
+	repoSpace := space.NewRepository(s.DB)
+	space, err := repoSpace.Create(context.Background(), &newSpace)
+	assert.Nil(t, err)
```

Comment:
Make this a `require.Nil(t, err)` because if there's an error during space creation, then it doesn't make any sense to continue.

Reply:
fixed it here 349dbde